test/integration/061_use_colors_tests/models/do_nothing_then_fail.sql

```diff
@@ -0,0 +1 @@
+with x as (select pg_sleep(1)) select 1,
```

Comment:
can this just `select 1 as id,` instead of calling `pg_sleep`? Unless I misunderstand, the sleep doesn't change the test in any way.

Reply:
yes it should work either way